qiskit_ibm_provider/ibm_backend.py

```diff
@@ -920,7 +922,9 @@ def _deprecate_id_instruction(
                 :meth:`IBMBackend.run()<IBMBackend.run>`. Modified in-place.
 
         Returns:
-            None
+            A mutated copy of the original circuit where 'id' instructions are replaced with
```

Comment:
```suggestion
            A modified copy of the original circuit where 'id' instructions are replaced with
```
At least to me, mutate implies modification in place which is what is being removed here, so maybe modified is better.